### PR TITLE
516 Computed

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-client/src/presenter/computeds/documentDetailHelper.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
   return {
-    isEditablePetition:
-      VALID_PETITION_METADATA_EDIT_STATUSES.indexOf(caseDetail.status) !== -1,
     showAction: (action, workItemId) => {
       const actions = get(state.workItemActions);
       return actions[workItemId] === action;
     },
+    isEditablePetition: ['New', 'Recalled'].includes(caseDetail.status),
+    showCaseDetailsView: caseDetail.status === 'Batched for IRS',
+    showCaseDetailsEdit: ['New', 'Recalled'].includes(caseDetail.status),
```
Comment (kkoskelin) [inline web-client/src/presenter/computeds/documentDetailHelper.js:11]: Would line 12 be a duplicate of line 10 here?  Are they used differently?

File: web-client/src/presenter/computeds/documentDetailHelper.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+    showServeToIrsButton: ['New', 'Recalled'].includes(caseDetail.status),
```
Comment (kkoskelin) [inline web-client/src/presenter/computeds/documentDetailHelper.js]: Can we then use this and get rid of `canServeToIRS` in caseDetailHelper?

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/computeds/documentDetailHelper.js (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/presenter/computeds/documentDetailHelper.js:11]: updated!

File: web-client/src/presenter/computeds/documentDetailHelper.js (hunk 2 above)
Comment (ericsorenson) [reply web-client/src/presenter/computeds/documentDetailHelper.js]: updated!

**Pull request merged**